fe/fe-core/src/main/java/com/starrocks/credential/hdfs/HDFSCloudCredential.java

```diff
@@ -23,31 +23,34 @@
 
 import java.util.Map;
 
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_AUTHENTICATION;
+
 public class HDFSCloudCredential implements CloudCredential {
-    public static final String EMPTY = "empty";
-    private String authentication;
+    public static final String SIMPLE_AUTH = "simple";
+    public static final String KERBEROS_AUTH = "kerberos";
+    protected String authentication;
     private String userName;
     private String password;
-    private String krbPrincipal;
-    private String krbKeyTabFile;
-    private String krbKeyTabData;
+    private String kerberosPrincipal;
+    private String keytab;
+    private String keytabContent;
```

Comment:
looks like the above three lines variable rename is not necessary, can revert back to keep this PR as focus as possible.